### PR TITLE
fix(navbar): prominent navbar for long titles

### LINKED ISSUE #9175: Navbar need ellipse and symbols limit
## Description  
If a long title is being added to navbar, e.g. 30+ symbols, and the title length exceeds the navbar length, the title goes on several rows. Check this example:
**https://stackblitz.com/edit/github-r2pf1i?file=src%2Fapp%2Fnavbar-sample-1%2Fnavbar-sample-1.component.html** 
The expected behavior is having a single-row title. Thus, ellipses and/or symbols limit to be implemented.
 
 * igniteui-angular version: 11.1.1
 * browser: all

### PR DESCRIPTION
**Pull request by SisIvanova**
Closes #9175 

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by SisIvanova** [comment]
material.io specification - https://material.io/components/app-bars-top#anatomy

**Comment by DaniMarinov** [comment]
As I have discussed with @SisIvanova, I suggest enlarging the navbar height if the title is too long, according to Material spec.

**Comment by StefanIvanov** [comment]
I find the approach to alleviate the concerns we have and also note that the [spec](https://material.io/components/app-bars-top#specs) shows two presets when it comes to height Regular at 56px and Extended at 128px

**Comment by StefanIvanov** [comment]
I don't find any issues and once @simeonoff verifies everything as well we can verify the proposed solution.

File: projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss [hunk 1]
```diff
@@ -83,6 +83,13 @@
     ));
 }
 
+@mixin line-clamp($lines, $inline, $vertical) {
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss:86]: This mixin should be a part of the mixins file in the utilities folder.

File: projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss [hunk 2]
```diff
@@ -83,6 +83,13 @@
...
+    display: if($inline, -webkit-inline-box, -webkit-box);
+    -webkit-line-clamp: if($lines, $lines, 'initial');
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss:88]:
```suggestion
    -webkit-line-clamp: if($lines, $lines, initial);
```

File: projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss [hunk 3]
```diff
@@ -83,6 +83,13 @@
...
+    -webkit-box-orient: if($vertical, 'vertical', 'initial');
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/navbar/_navbar-theme.scss:89]:
```suggestion
    -webkit-box-orient: if($vertical, vertical, initial);
```

**Review by simeonoff — CHANGES_REQUESTED** [review]

**Review by simeonoff — APPROVED** [review]

**Pull request merged**